catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ImportCommand.java

```diff
@@ -148,6 +153,7 @@ protected Object executeWithSubject() throws Exception {
                     } catch (IOException | CatalogTransformerException e) {
                         LOGGER.debug("Could not transform metacard: {}", id);
                     }
+                    metacard = applyInjectors(metacard, attributeInjectors);
```

Comment:
This import command is not using the CatalogFramework to create metacards, it is putting them back straight through the provider.